prac_07/project_management.py

```diff
@@ -0,0 +1,140 @@
+from project import Project
+import datetime
+
+PROJECTS_FILE = "projects.txt"
```

Comment:
Not sure if you should use global variables here. Perhaps taking the file name as an input might be more suitable?